### ISSUE TITLE
`@stylistic/declaration-colon-space-after` false positive with `!important`

### ISSUE THREAD

**Issue by VChet** [stylelint-stylistic/stylelint-stylistic#109]
### What minimal example or steps are needed to reproduce the bug?

A custom property with comment as its value produces no errors as expected:

```css
a {
  --a: /*comment*/;
}
```

Adding `!important` causes an error:

```css
a {
  --a: /*comment*/ !important;
}
```
`Expected single space after ":" with a single-line declaration`

I was using this syntax  for `@advanced` variables in UserCSS, as mentioned [in the Stylus wiki](https://github.com/openstyles/stylus/wiki/Writing-UserCSS#more-examples)

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": "@stylistic/stylelint-plugin",
  "rules": {
    "@stylistic/declaration-colon-space-after": "always"
  }
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYAdAdilBaXS8KA9AFSQC2FMmALqcQNxYC+WWqG2eBRZl1OgxQBCAJYUADhABOtJHWaY2mEABoQAMzEAbGADkk1BCBgAPI5L0A6MAGc768BEzaA5ifPS5KWJqQArjq0aFg46CBWAW5imHYRRBEAAna0AJ46YqliYMSpGTCZdLhRMZgRamEoETJBMPEIodzhICnpmdm5sGA6SDJItGIuuJA6w3aSSGAwBJq0MDIJ1cg6AO5IaQ1VKixKTpCuYm4AYrIUAyb5ekW0ti7u1hQAVo4asJKOiFwtbRlZg7kroVYrQRvcjksIr15qkKlUIkCbpCQND6rQIqwsCAWEA)

### Which Stylelint-related dependencies are you using?

N/A

### What did you expect to happen?

No errors should be reported. The single space after `:` should be recognized correctly by `@stylistic/declaration-colon-space-after: "always"`.

### What actually happened?

The error is reported: `Expected single space after ":" with a single-line declaration`

Using `--fix` adds another space after `:`
```diff
- --a:·/*comment*/ !important;
+ --a:··/*comment*/ !important;
```
The declaration is still reported as an error.

### Do you have a proposal to fix the bug?

https://github.com/stylelint-stylistic/stylelint-stylistic/pull/110

**Comment by firefoxic** [comment]
@VChet, hi!

Thanks for your report!
